setup.py

```diff
@@ -13,7 +13,7 @@
 from setuptools import find_packages, setup  # noqa: H301
 
 NAME = "marquez-python"
-VERSION = "0.1.5"
+VERSION = "0.1.7"
```

Comment:
Aren't we bumping the version to `0.1.6`?

Reply:
There was a PR that didn't get merged that would have bumped it to 0.1.6, but it's 0.1.5 in the PyPi (though version is marked as 1.6). Since I can't re-submit the same version (due to PyPi filename rule), I want to bypass this and set both to 0.1.7.

Reply:
I see. Yeah, that's not great and we'll want to automate this going forward. I opened an issue to capture the change in our release flow: https://github.com/MarquezProject/marquez-python/issues/16